### ISSUE TITLE
No sorting options

### ISSUE THREAD

**Issue by Niremizov** [deecay/redmine_pivot_table#28]
While configuring pivottable I cannot find sort options, isn't they implemented?

**Comment by Niremizov** [comment]
Sorting is not supported by PivotTable.js https://github.com/nicolaskruchten/pivottable/wiki/Frequently-Asked-Questions#sorting-in-built-in-table-renderer

But probably we can sort dataset itself by changing query?

**Comment by deecay** [comment]
No, I think sorting has to be added to the table with Javascript, after HTML table is rendered. I'm not thinking about implementing this feature for now.

See https://www.nrecosite.com/pivot_table_aspnet.aspx for example.
